examples/README.md

```
@@ -38,34 +38,42 @@ Central index of runnable example scripts demonstrating Mailtrap PHP SDK feature
 | Message CRUD / listing | [`testing/messages.php`](testing/messages.php) |
 | Project CRUD / listing | [`testing/projects.php`](testing/projects.php) |
 
-### 4. Contact Management
+### Inbound Email
+| Purpose | File |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)